packages/ketcher-react/src/script/render/restruct/index.js

```diff
@@ -616,8 +616,21 @@ ReStruct.prototype.setSelection = function (selection) {
   })
 }
 
+/**
+ * SelectionPlate could be an item then value would be in it
+ * or it could be a set of items then removed value need to be check on at least one of items in set
+ * @param item
+ * @returns {boolean}
+ */
+function isSelectionSvgObjectExists(item) {
+  return (
+    (item.selectionPlate !== null && !item.selectionPlate.removed) ||
+    (item.selectionPlate instanceof Array && !item.selectionPlate[0].removed)
```

Comment:
better change
`item.selectionPlate instanceof Array` to `Array.isArray(item.selectionPlate)`

Reply:
fixed